code/__DEFINES/is_helpers.dm

```diff
@@ -179,3 +179,6 @@
 
 // Shuttles
 #define isshuttleturf(T) (length(T.baseturfs) && (/turf/baseturf_skipover/shuttle in T.baseturfs))
+
+// Xeno hives
+#define isnormalhive(hive) (istype(hive, /datum/hive_status/normal))
```

Comment:
Just implement my solution from #2469 regarding hives instead of making a hacky helper, adding such functionality to the rest of the hives is a good thing.

Reply:
Dejavu.

Reply:
Hehe